lib/emscripten/src/README.md

```diff
@@ -385,7 +385,7 @@
   ```
 - **read** (\_\_\_syscall3) ✅ ❗️ &nbsp;&nbsp;&nbsp;&nbsp;[:top:](#host-apis)
   ```rust
-  fn read(fd: c_int, buf: u32, count: size_t, instance: &mut Instance) -> ssize_t
+  fn read(fd: c_int, buf: u32, count: size_t, vmctx: &mut Instance) -> ssize_t
```

Comment:
In the docs, you should change all `&mut Instance` to `&mut Ctx`.

Reply:
Noted.